python/py-circuitbreaker/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-circuitbreaker
+version             2.0.0
+
+license             BSD-3-Clause
+platforms           {darwin any}
+supported_archs     noarch
+maintainers         nomaintainer
+
+description         Python implementation of the \"Circuit Breaker\" Pattern
+long_description    {*}${description}
+
+homepage            https://github.com/fabfuel/circuitbreaker
+
+checksums           md5     0fc04e58267a46ac371aac0f3e2b0105 \
+                    rmd160  992822cb56baf69b33a3da002c9ebbf5dfa56ddd \
+                    sha256  28110761ca81a2accbd6b33186bc8c433e69b0933d85e89f280028dbb8c1dd14 \
+                    size    10736
+
+python.rootname     circuitbreaker
+python.versions     37 38 39 310 311
```

Comment:
please don't add EOL Python versions unless absolute requires, which seems unlikely here as this is a new port. In other words, remove `37`